### PR TITLE
Maintenance Week: add tests & story for Zooniverse Logo & Logotype

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview

Follows #3750 
Package: lib-react-components
Part of: Maintenance Week

This PR adds React Testing Library tests for Zooniverse Logotype, and stories for both Zooniverse Logo & Logotype.

Ready for review

### CODE REVIEW

File: packages/lib-react-components/src/ZooniverseLogo/ZooniverseLogo.stories.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { Box, Button, Grommet } from 'grommet'
+
+import readme from './README.md'
+import ZooniverseLogo from './ZooniverseLogo'
+
+export default {
+  title: 'Components/ZooniverseLogo',
+  component: ZooniverseLogo,
+  args: {
+    id: 'The Zooniverse',
```
Comment (eatyourgreens) [inline packages/lib-react-components/src/ZooniverseLogo/ZooniverseLogo.stories.js:11]: Should this be a HTML element `id`, in which case 'The Zooniverse' is not a valid identifier.

**Review by eatyourgreens — APPROVED** [review]
LGTM. `id` might need to be [a valid identifier](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) so double check that.

**Review by eatyourgreens — CHANGES_REQUESTED** [review]
I checked and those IDs should be DOM element identifiers, so no spaces allowed!

**Comment by shaunanoordin** [comment]
Thanks! 👍 For both stories, the `id: 'The Zooniverse'` has now been changed to `id: 'the-zooniverse'`

**Review by eatyourgreens — APPROVED** [review]

**Pull request merged**